keepa/interface.py

```diff
@@ -1866,6 +1866,11 @@ def product_finder(self, product_parms, domain='US'):
             One of the following Amazon domains: RESERVED, US, GB, DE,
             FR, JP, CA, CN, IT, ES, IN, MX Defaults to US.
 
+        include_details : bool, optional
+            If False, returns just the list of product IDs.  If True, returns the full
+            response payload, including the 'totalResults' field which allows the caller
+            to know whether or not the full list of products was retrieved.
```

Comment:
Would be worth briefly documenting how this flag changes the shape of the returned dictionary.

Reply:
Agreed.